help/sources/tutorials/ui/create/analytics/rainfocus-experience.md

```diff
@@ -0,0 +1,202 @@
+| session.title             | String   |                Inspiration Keynote                |                       Title of session                       |
+| session.length            | Int      |                         90                        |                       Length of session                      |
+| sessiontime.sessiontimeId | String   |                1673033149739001OJLZ               |           RainFocus Identifier for the session time          |
+| sessiontime.startTime     | String   |                2023-03-22 10:00:00                |                      Session start time                      |
+| sessiontime.endTime       | String   |                2023-03-22 10:00:00                |                       Session end time                       |
+| sessiontime.room          | String   |                        B32                        |                       Room for session                       |
+
+To use the sample Schema from RainFocus, you may take one of the the following steps:
+
+* [Create the Schema using the UI](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-ui.html?lang=en)
+* [Create the Schema using the API](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-api.html?lang=en)
+
+>[!IMPORTANT]
+>
+>* The schema must extend the **XDM ExperienceEvent class.** 
+> * You must ensure that the Schema includes a **primary identity**, and is **enabled for Profile**
+>   * [Define Identity Fields in the UI](https://experienceleague.adobe.com/docs/experience-platform/xdm/ui/fields/identity.html?lang=en)
+>* You may substitute the example identity (Email) for another appropriate identifier such as a sha256 email or ECID.
+
+## Enable the Integration Profile in RainFocus
+
+Now that the Service Account and XDM Schema are ready, switch to RainFocus to activate the Integration Profile, which will be responsible for streaming data to AEP. 
+
+1. Log into the **RainFocus** platform
+
+2. In the primary navigation, select **Libraries** and **Integration Profiles** 
+![Screenshot showing the RainFocus platform and the location of the Integration Profile in the main menu](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile.png)
+
+3. Select the **+** icon, in the bottom right corner, to create a new profile.
+
+4. Choose the **Adobe Experience Platform CDP** and click **OK**  
+![Screenshot showing the RainFocus platform and the location of the AEP CDP Integtaion Profile](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-select.png)
+
+5. Copy the following credential, previously created in the Adobe Developer Portal Project
+
+   1. **Client ID**
+   2. **Client Secret**
+   3. **Technical Account ID**
+   4. **Organization ID**
+
+6. Click **Save.** You should now see the new Integration Profile listed.
```

Comment:
We can combine this to something like:
"Provide the credentials that were previously created in the Adobe Developer Portal Project:
   1. **Client ID**
   2. **Client Secret**
   3. **Technical Account ID**
   4. **Organization ID**
Then, select **Save.** You should now see the new Integration Profile listed."